crates/cargo-contract/src/cmd/extrinsics/info.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018-2023 Parity Technologies (UK) Ltd.
```

Comment:
This command should be moved out of the `extrinsics` folder

Reply:
It is already in progress